src/main/java/Duke.java

```diff
@@ -9,9 +9,11 @@ public static void main(String[] args) {
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
 
-        String[] texts = new String[100];
+        Task[] texts = new Task[100];
         String line;
+        String secondline;
```

Comment:
The naming convention for variables is camelCase, so do remember to use the correct naming convention!

Reply:
similar for all the variables, eg NumberOfTexts should be numberOfTexts